packages/venia-concept/src/components/Checkout/form.js

```diff
@@ -177,6 +180,7 @@ class Form extends Component {
                     </Section>
                 </div>
                 <div className={classes.footer}>
+                    <ResetButton resetCheckout={resetCheckout} />
```

Comment:
After looking at the [ResetButton](https://github.com/magento-research/pwa-studio/blob/release/2.0/packages/venia-concept/src/components/Checkout/resetButton.js) component I feel like it's an extra abstraction that we don't need. Can we delete that component entirely and just implement this as  `<Button onClick={resetCheckout}>Continue Shopping</Button>`?